src/ElmSharp/ElmSharp/Window.cs

```diff
@@ -1311,5 +1311,27 @@ static void Preload()
         /// For internal use only
         /// </summary>
         internal static Window CreateWindow(string name) => PreloadedWindow.GetInstance() ?? new Window(name);
+
+        /// <summary>
+        /// Disable window effect. For internal use only.
+        /// </summary>
+        internal static void DisableWindowEffect(Window window)
+        {
+            if (window != null)
+            {
+                Interop.Elementary.elm_win_aux_hint_add(window.Handle, "wm.policy.win.effect.disable", "1");
+            }
+        }
+
+        /// <summary>
+        /// Enable window effect. For internal use only.
+        /// </summary>
+        internal static void EnableWindowEffect(Window window)
+        {
+            if (window != null)
+            {
+                Interop.Elementary.elm_win_aux_hint_add(window.Handle, "wm.policy.win.effect.disable", "0");
+            }
+        }
```

Comment:
Instead of tightly coupled with specific value (`wm.policy.win.effect.disable`), how about providing a method that can have parameter like the native platform for flexibility?

e.g. 
```cs
Window.AddAuxiliaryHint(string key, string value)
```

Or, if you want keep current API signature and if this change is only meaningful for wearable profile, how about adding it to ElmSharp.Wearable?

Reply:
Good Idea. I added all window auxiliary hint APIs.